ukvm/ukvm_module_gdb.c

```diff
@@ -332,25 +230,22 @@ unsigned char *getpacket(void)
         buffer[count] = 0;
 
         if (ch == '#') {
-            ch = getDebugChar();
+            ch = recv_char();
             xmitcsum = hex(ch) << 4;
-            ch = getDebugChar();
+            ch = recv_char();
             xmitcsum += hex(ch);
 
             if (checksum != xmitcsum) {
-                if (remote_debug) {
-                    fprintf(stderr,
-                            "bad checksum.  My count = 0x%x, sent=0x%x. buf=%s\n",
-                            checksum, xmitcsum, buffer);
-                }
-                putDebugChar('-');        /* failed checksum */
+                errx(1, "bad checksum.  My count = 0x%x, sent=0x%x. buf=%s",
```

Comment:
Maybe a more descriptive error message? "Failed checksum from GDB"? Also, you then send a response so should this be a `warnx` instead?